### DISCUSSION TITLE
Starting JAX without doing anything can take up 10GB GPU  memory, is it normal?

### DISCUSSION THREAD

**Discussion by hbb1** [jax-ml/jax#6909] [General] [answered]
```python 
import jax 
jax.random.PRNGKey(0)
```
After I run these two lines of code, I found my GPU memory increases 10GB? It is normal?

**Comment by hawkinsp** [comment, marked as answer]
Yes, this is expected and working as intended. See https://jax.readthedocs.io/en/latest/gpu_memory_allocation.html

**Reply by hbb1** [reply]
I see, thank you.
